prow/cmd/gerrit-crier/main.go

```diff
@@ -45,15 +46,25 @@ const (
 )
 
 type options struct {
-	numWorkers int
-	masterURL  string
-	kubeConfig string
+	numWorkers     int
+	masterURL      string
+	kubeConfig     string
+	cookiefilePath string
+	projects       gerritutil.ProjectsFlag
 }
 
 func (o *options) Validate() error {
 	if o.numWorkers == 0 {
 		return errors.New("--num-workers must be greater than 0")
 	}
+
+	if len(o.projects) == 0 {
+		return errors.New("--gerrit-projects unset")
```

Comment:
nit: maybe "must be set" to match the other error? nbd though

Reply:
hard to unify all three errors... the above one actually has a default value of 1 🙃

Reply:
(also it matches with https://github.com/kubernetes/test-infra/blob/master/prow/cmd/gerrit/main.go#L68-L86)

Reply:
imo "error foo unset" doesn't really call out that we expect it to be set by the user as strongly as "must be set", but not a big deal. imo that can more easily be read as some internal logic failed to set it. /shrug

Reply:
changed in both places